bundles/org.eclipse.passage.loc.billing.core/META-INF/MANIFEST.MF

```diff
@@ -7,3 +7,5 @@ Bundle-Name: %Bundle-Name
 Bundle-Vendor: %Bundle-Vendor
 Bundle-Copyright: %Bundle-Copyright
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Import-Package: org.eclipse.passage.lic.users
```

Comment:
Now we requires the full bundle, not a package. Let's change to "Required-Bundle" for consistency